.github/workflows/backup-cognito-users.yaml

```diff
@@ -0,0 +1,44 @@
+name: Create backup of cognito users
+on:
+  workflow_dispatch:
```

Comment:
this should run automatically every night or week instead of manually right? see for reference https://github.com/hms-dbmi-cellenics/testing/blob/master/.github/workflows/run-nightly-e2e.yaml

Reply:
yes, this should run automatically once per week, ideally during time when nobody is around

Reply:
The automation of this github action is done in this ticket :https://biomage.atlassian.net/browse/BIOMAGE-1728 together with implementing notifications and error checking

Reply:
I think that considering that the original ticket was completely replaced by creating the action it makes sense to do all of this together because it's almost no extra work. I've already discussed this with stefan